bindings/python/pinocchio/robot_wrapper.py

```diff
@@ -139,28 +127,12 @@ def frameClassicAcceleration(self, index):
         a.linear += np.cross(v.angular, v.linear, axis=0)
         return a;
 
-    @deprecated("This method is now deprecated. Please use jointJacobian instead. It will be removed in release 1.4.0 of Pinocchio.")
-    def jacobian(self, q, index, update_kinematics=True, local_frame=True):
-        if local_frame:
-            return pin.jointJacobian(self.model, self.data, q, index, pin.ReferenceFrame.LOCAL, update_kinematics)
-        else:
-            return pin.jointJacobian(self.model, self.data, q, index, pin.ReferenceFrame.WORLD, update_kinematics)
-
-    def jointJacobian(self, q, index, *args):
-        if len(args)==0:
-            return pin.jointJacobian(self.model, self.data, q, index)
-        else: # use deprecated signature (19 Feb 2019)
-            update_kinematics = True if len(args)==1 else args[1]
-            rf = args[0]
-            return pin.jointJacobian(self.model, self.data, q, index, rf_frame, update_kinematics)
+    def jointJacobian(self, q, index):
+        return pin.jointJacobian(self.model, self.data, q, index)
 
     def getJointJacobian(self, index, rf_frame=pin.ReferenceFrame.LOCAL):
         return pin.getFrameJacobian(self.model, self.data, index, rf_frame)
```

Comment:
There is a bug here! It should call `getJointJacobian`! Apparently, it was there before! And, judging from `git blame`, it was *me* doing this mistake!